sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_transport_async.py

```diff
@@ -486,9 +483,9 @@ async def connect(self):
             # https://github.com/aio-libs/aiohttp/pull/5860
             # https://github.com/aio-libs/aiohttp/issues/2309
 
-            self.ws = await self.session.ws_connect(
+            self.sock = await self.session.ws_connect(
                 url=url,
-                timeout=self._connect_timeout,
+                timeout=self.connect_timeout,    # timeout for connect
```

Comment:
We used `socket_timeout` for the sync websocket timeout? I know they're both `1` by default, but from a design perspective it would be good to be consistent.